### PR TITLE
docs: draw.io guide

### PR DESCRIPTION
**Pull request by Sandy-at-Tazama**
#19 Adding new draw.io guide and images
#49 docs: Update contribution guide SPDX-License-Identifier: Apache-2.0

### CODE REVIEW

File: Community/Tazama-Contribution-Guide.md [hunk 1]
```diff
@@ -486,6 +484,8 @@ Best practices for making changes.
 
 **Code Documentation** – Electronic documentation has been auto-generated (where available) and has been checked for correctness. Also have sufficient code comments. [Developer] + [PR Approver]
 
+**Licensing comment** Add the following string as a comment at the top of every file in the FRMS organisation in GitHub that is capable of including a comment  " SPDX-License-Identifier: Apache-2.0"
```
Comment (Justus-at-Tazama) [inline Community/Tazama-Contribution-Guide.md:487]: I'd specify the different files and comment formats explicitly here. Technically, JSON files don't typically get commented but we've added the header to ours, so we'll need to tell people explicitly to make sure they add it there too.

File: Guides/full-service-full-stack-docker-tazama .md [hunk 2]
```diff
@@ -4,12 +4,6 @@
 - [Pre-requisites:](#pre-requisites)
 - [Installation steps](#installation-steps)
 - [1. Clone the Full-Stack-Docker-Tazama Repository to Your Local Machine](#1-clone-the-full-stack-docker-tazama-repository-to-your-local-machine)
-- [2. Update the Full-Stack-Docker-Tazama Configuration Files](#2-update-the-full-stack-docker-tazama-configuration-files)
```
Comment (Justus-at-Tazama) [inline Guides/full-service-full-stack-docker-tazama .md:7]: Were these headings removed intentionally?

File: Guides/full-service-full-stack-docker-tazama .md [hunk 3]
```diff
@@ -41,183 +35,13 @@ This guide is specific to the Windows 10 operating system.
...
-        set GH_TOKEN
-        ```
-
-[Top](#introduction)
 
 ## Installation steps
 
 ## 1. Clone the Full-Stack-Docker-Tazama Repository to Your Local Machine
 
-In a Windows Command Prompt, navigate to the folder where you want to store a copy of the source code. Then clone the repository with the following command:
```
Comment (Justus-at-Tazama) [inline Guides/full-service-full-stack-docker-tazama .md:70]: Also these?

**Review by Justus-at-Tazama — CHANGES_REQUESTED** [review]
Can we please review the changes to the full-stack guide?

File: Community/Tazama-Contribution-Guide.md (hunk 1 above)
Comment (Sandy-at-Tazama) [reply Community/Tazama-Contribution-Guide.md:487]: Can you help with the different file types and comment formats?

File: Guides/full-service-full-stack-docker-tazama .md (hunk 2 above)
Comment (Sandy-at-Tazama) [reply Guides/full-service-full-stack-docker-tazama .md:7]: No!  How do I undo?

Comment (Sandy-at-Tazama) [reply Guides/full-service-full-stack-docker-tazama .md]: restored

File: Community/Tazama-Contribution-Guide.md (hunk 1 above)
Comment (Sandy-at-Tazama) [reply Community/Tazama-Contribution-Guide.md]: File types for License header added to contribution guide

**Pull request merged**